.github/workflows/terraform-validate.yml

```diff
@@ -21,6 +21,12 @@ jobs:
       with:
         terraform_version: 1.8.4
 
+    - name: Initialize Terraform
+      id: init
+      run: |
+        cd my-terraform-project
+        terraform init
+
```

Comment:
To run terraform validate, it is not necessary to run terraform init beforehand.